### PR TITLE
[Admin][Product] Add view in store button

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-admin-panel
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

Screenshot 2024-05-17 at 07 33 52

Screenshot 2024-05-17 at 07 33 34

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Product/ShowPage.php [hunk 1]
```diff
@@ -81,13 +81,13 @@ public function showVariantEditPage(ProductVariantInterface $variant): void
...
     {
         return array_merge(parent::getDefinedElements(), [
+            'breadcrumb' => '.breadcrumb > div',
             'edit_product_button' => '#edit-product',
             'edit_variant_button' => '#variants .variants-accordion__title:contains("%variantCode%") .edit-variant',
             'product_name' => '#header h1 .content > span',
-            'show_product_dropdown' => '.scrolling.menu',
-            'show_product_single_button' => '.ui.labeled.icon.button',
+            'show_product_dropdown' => '[data-test-view-in-store]',
+            'show_product_single_button' => '[data-test-view-in-store]',
```
Comment (GSadee) [inline src/Sylius/Behat/Page/Admin/Product/ShowPage.php]: As they have the same value:
```suggestion
            'show_product_button' => '[data-test-view-in-store]',
```

File: src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php [hunk 2]
```diff
@@ -365,8 +365,8 @@ protected function getDefinedElements(): array
                 'product_taxon' => '#sylius-product-taxonomy-tree .item .header:contains("%taxonName%") input',
                 'product_taxons' => '#sylius_product_productTaxons',
                 'shipping_required' => '#sylius_product_variant_shippingRequired',
-                'show_product_dropdown' => '.scrolling.menu',
-                'show_product_single_button' => '[data-test-show-product-in-shop-page]',
+                'show_product_dropdown' => '[data-test-view-in-store]',
+                'show_product_single_button' => '[data-test-view-in-store]',
```
Comment (GSadee) [inline src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php]: As above 🤔

File: src/Sylius/Bundle/AdminBundle/templates/product/common/header/title_block/actions/view_in_store.html.twig [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon as _icon %}
+
+{% set product = (hookable_metadata.context.resource) %}
+{% set enabledChannels = product.channels|filter(channel => channel.enabled == true) %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/product/common/header/title_block/actions/view_in_store.html.twig]: Use snake_case for twig variables:
```suggestion
{% set enabled_channels = product.channels|filter(channel => channel.enabled == true) %}
```

File: src/Sylius/Bundle/AdminBundle/templates/product/show/content/header/title_block/title.html.twig [hunk 4]
```diff
@@ -5,3 +5,6 @@
         <h1 class="page-title">{{ product.name }}</h1>
     </div>
 </div>
+<div class="col-12 col-md-auto ms-auto d-print-none">
+    {% hook 'actions' %}
+</div>
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/AdminBundle/templates/product/show/content/header/title_block/title.html.twig]: Pretty sure this is unnecessary, but if so it's in the wrong place, it's a part of the title block, but not of the title itself.

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @mpysiak!

**Pull request merged**